src/test_pages/TimelinePage.ts

```diff
@@ -22,24 +25,62 @@ export default class TimelinePage extends BasePage {
     this.operators_dropdown = this.page.locator("button[aria-label='Open']")
     this.operators_list = this.page.locator('ul#operator-select-listbox')
     this.eged_taavura = this.page.locator("//li[text()='אגד תעבורה']")
-    this.station_select = this.page.locator('')
+    this.station_select = this.page.locator('#stop-select')
+    this.routes_list = this.page.locator('ul#route-select-listbox')
+    this.stop_station_list = this.page.locator('ul#stop-select-listbox')
+    this.timeline_graph = this.page.locator('h4')
   }
 
-  public async selectOperatorFromDropbox(operatorName: string) {
-    await this.selectFrom_UL_LI_Dropbox(this.operators_dropdown, this.operators_list, operatorName)
+  public async selectOperatorFromDropbox(locator: Locator, list: Locator, operatorName: string) {
+    await this.selectFrom_UL_LI_Dropbox(locator, list, operatorName)
   }
 
   public async fillLineNumber(lineNumber: string) {
     await this.fillTextToElement(this.line_number, lineNumber)
   }
 
   public async closeLineNumber() {
-    await this.clickOnElement(this.close_line_number)
+    await this.clickOnElement(this.close_line_number, 3000)
   }
 
-  public async verifyRouteSelectionVisible(isVisible: boolean) {
-    isVisible
-      ? await expect(this.route_select).toBeVisible({ timeout: 2000 })
-      : await expect(this.route_select).toBeHidden({ timeout: 2000 })
+  public async verifyRouteSelectionVisible(locator: Locator, isVisible: boolean, timeout?: number) {
+    await this.verifySelectionVisible(locator, isVisible, timeout)
+  }
+
+  public async verifyDuplications(selectBox: Locator) {
+    const list = new Set()
+    const selectOption = await this.getAllOptions_Dropbox(selectBox)
+    for (const row of selectOption) list.add(await row.textContent())
+    expect(list.size === selectOption.length).toBe(true)
+  }
+
+  public async verifyLineNumberNotFound() {
+    await expect(this.page.getByText('הקו לא נמצא')).toBeVisible()
+  }
+
+  public get routeSelect() {
+    return this.route_select
+  }
```

Comment:
```suggestion
  public get routeSelect() {
    return this.page.locator('#route-select')
  }
```

this way, you won't need `route_select` at all - you can delete the constructor.
@DanLeshinsky